firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift

```diff
@@ -531,6 +531,12 @@ class NavigationTest: BaseTestCase {
     private func validateExternalLink(isPrivate: Bool = false) {
         navigator.openURL("ultimateqa.com/dummy-automation-websites")
```

Comment:
Wondering if we could use a custom page that we manage so that we don't relay on external sites or as less as possible..  cc @dragosb01 @mdotb-moz

Reply:
When the test was created i looked over all the custom pages and none had external links.